modo/remote.py

```diff
@@ -0,0 +1,52 @@
+"""Functions related to server storage handling"""
+
+from pydantic import HttpUrl
+import requests
+from typing import List, Mapping, Optional
+
+
+def list_remote_items(remote_url: HttpUrl) -> List[HttpUrl]:
+    return requests.get(url=remote_url + "/list").json()
+
+
+def get_metadata_from_remote(
+    remote_url: HttpUrl, id: Optional[str] = None
```

Comment:
**praise:** simple yet very useful :D
**suggestion:** avoid shadowing built-in `id` (`id_`, `modo_id`, ...)